.gitignore

```diff
@@ -24,6 +24,9 @@ Win32-Release/
 x64-Debug/
 x64-Release/
 
+# Eclipse
```

Comment:
Please revert the changes to this file. This really belong in a global file <https://docs.github.com/en/github/using-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer> otherwise this file will just be a collection of IDE specific directories.

Reply:
reverted the .gitignore.  Putting the IDE stuff in a global git ignore is a really good idea.  My company just shoves them in our local repos, so I was used to that.  I will use your approach going forward.  Thanks!